pkg/api/handlers_data.go

```diff
@@ -100,7 +100,7 @@ func handleWorkResponse(c *gin.Context, responseCh chan []byte) {
 
 			c.JSON(http.StatusOK, result)
 			return
-		case <-time.After(90 * time.Second):
+		case <-time.After(60 * time.Second):
```

Comment:
we should update this to 10s for now to be very aggressive with the timeout response time @jdutchak @restevens402 - I will push a commit for this